x-pack/plugins/ui_actions_enhanced/public/drilldowns/drilldown_definition.ts

```diff
@@ -31,11 +30,8 @@ import { PersistableStateDefinition } from '../../../../../src/plugins/kibana_ut
 
 export interface DrilldownDefinition<
   Config extends BaseActionConfig = BaseActionConfig,
-  SupportedTriggers extends TriggerId = TriggerId,
-  FactoryContext extends BaseActionFactoryContext<SupportedTriggers> = {
-    triggers: SupportedTriggers[];
-  },
-  ExecutionContext extends TriggerContextMapping[SupportedTriggers] = TriggerContextMapping[SupportedTriggers]
+  ExecutionContext extends object = object,
+  FactoryContext extends BaseActionFactoryContext = BaseActionFactoryContext
```

Comment:
I switched the order of ExecutionContext and FactoryContext since there were a few use cases where the default for ExecutionContext needed to be overridden but not FactoryContext